props/prop_base.h

```diff
@@ -607,7 +622,11 @@ class PropBase : CommandParser, Looper, protected SaberBase {
   void PollScanId() {
     if (find_blade_again_pending_) {
       find_blade_again_pending_ = false;
+      blade_id_should_report_= true;
       FindBladeAgain();
+      // Indicate what happened when different Blade ID detected
+      SaberBase::DoNewFont();
+      STDOUT << "DISPLAY: " << current_preset_name() << "\n";
```

Comment:
Why does this say DISPLAY?
Why is it needed?
Why isn't it a PVLOG?

Reply:
DISPLAY was there already (well not HERE , but I reused this line)
See above for why.
Should be PVLOG, yes. Again, i just reused the line.

Reply:
updated.
SetPreset()'s  line made PVLOG